Assets/MixedRealityToolkit.Tests/PlayModeTests/TestFixture_02_ManipulationTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// Unity doesn't include the the required assemblies (i.e. the ones below).
+// Given that the .NET backend is deprecated by Unity at this point it's we have
+// to work around this on our end.
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using System.Collections;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using Microsoft.MixedReality.Toolkit;
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class TestFixture_02_ManipulationTests
```

Comment:
I see that this follows the edit mode naming scheme - do you know if the TestFixture and # actually is necessary here? If not, they are arbitrary and non-value accruing noise (i.e. the numbering is arbitrary based on when things are being added, and TestFixture prefixes everything and isn't super helpful). If this works without the naming, I would rather we move our tests to be named based on what they are testing.

Reply:
Yeah, I'm not aware of the naming being a requirement. TestFixture is NUnit's term, and it's an optional attribute that can be added to the class.

If anything, the numbering helps organize the test fixtures in the Test Runner window in a specific order (instead of alphabetically).

Reply:
Removed the TestFixture_ prefxi from PlayMode tests. I think we could also remove them from EditoMode Tests.